### PR TITLE
Additional service worker tests for custom responses

### PR DESCRIPTION
**Pull request by jakearchibald**
Created these after noticing a couple of Safari bugs. It fails the 'delayed' stream tests, and the blob response tests.

+@beidson

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/22096)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/319679441)

File: service-workers/service-worker/resources/fetch-event-respond-with-custom-response-worker.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+addEventListener('fetch', event => {
+  const url = new URL(event.request.url);
+  const type = url.searchParams.get('type');
```
Comment (wanderview) [inline service-workers/service-worker/resources/fetch-event-respond-with-custom-response-worker.js]: Does edge support search params now?  I guess we might use them in other SW tests already.

**Review by wanderview — APPROVED** [review]
LGTM assuming URLSearchParams are ok with edge.

**Comment by beidson** [comment]
If there’s a SW WPT we fail, we are aware and we’re working on it :)

**Comment by annevk** [comment]
I went ahead and merged this based on @wanderview's review. Hope that's okay.

**Comment by jakearchibald** [comment]
Cheers!

**Pull request merged**